src/applications/vaos/referral-appointments/utils/provider.js

```diff
@@ -47,7 +49,8 @@ const providers = {
  * @returns {Object} Provider object
  */
 const createProviderDetails = (numberOfSlots, providerId = '111') => {
-  const provider = providers[providerId];
+  // Deep copy the provider object to avoid modifying the original during tests
+  const provider = JSON.parse(JSON.stringify(providers[providerId]));
```

Comment:
I copied in the return in my PR and it seemed to solve this issue. I don't mind this approach either. But one should win out. We can chat about it a bit tomorrow if you are in.

Reply:
Agree! I don't have a preference either but this was just a quick and dirty solution after spending a few hours wondering how the test were failing that should not be. Let's sync up tomorrow about it

Reply:
Used the fix from your PR and it seemed to work

Reply:
oh nice!